x-pack/plugins/enterprise_search/public/applications/app_search/icons/cursor_icon.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+export const CursorIcon: React.FC = ({ ...props }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    className="euiIcon"
+    width={16}
+    height={16}
+    {...props}
```

Comment:
What custom props do you anticipate this icon needing in the future? It looks like we're not currently passing in any custom props in this PR, so I might suggest leaving it out if it's not being used:

```suggestion
export const CursorIcon: React.FC = () => (
  <svg
    xmlns="http://www.w3.org/2000/svg"
    className="euiIcon"
    width={16}
    height={16}
```

Reply:
I can't eloquently explain why, but the inclusion of `{...props}` here allows us to set the size of the icon and, IIRC, ensures the icon renders properly in dark mode.

Reply:
Ohh you're totally right, apologies, I hadn't realized the interaction between `CursorIcon` and `EuiIcon` in AnalyticsHeader, what tripped me up was looking for a standard `<CursorIcon />` type component/usage. Cool, disregard/collapsing this comment